crates/sui-core/src/overload_monitor.rs

```diff
@@ -3,12 +3,19 @@
 
 use crate::authority::AuthorityState;
 use std::cmp::{max, min};
+use std::hash::Hasher;
 use std::sync::atomic::{AtomicBool, AtomicU32, Ordering};
 use std::sync::Weak;
 use std::time::Duration;
+use std::time::{SystemTime, UNIX_EPOCH};
 use sui_config::node::OverloadThresholdConfig;
+use sui_types::digests::TransactionDigest;
+use sui_types::error::SuiError;
+use sui_types::error::SuiResult;
+use sui_types::fp_bail;
 use tokio::time::sleep;
 use tracing::{debug, info};
+use twox_hash::XxHash64;
```

Comment:
We discussed using other hash functions, but I still decide to use xxhash because it seems that xxhash is recommended more often than siphash when I search.